src/components/RepoWrap.tsx

```diff
@@ -43,20 +43,17 @@ const RepoWrap = (): JSX.Element => {
 
   return (
     <div className="bg-darkestGrey">
-      <SecondaryNav
-        activeLink={activeLink}
-        user={user}
-      />
+      <SecondaryNav activeLink={activeLink} user={user} />
 
       <HotRepositories />
 
-      {!isLoading &&
-        <ListRepositories
-          activeLink={activeLink}
-          fetchedData={data}
-          handleLoadingMore={handleLoadingMore}
-          limit={limit}
-        />}
+      <ListRepositories
```

Comment:
scratch that earlier comment, I now see you are passing loading in here - it should take error too tho :D

Reply:
What should we return if there is an error? 
```ts
// ListRepositories.tsx
// temporary solution
if(error) {
  console.error(error)
}
```